docs/source/tutorials/interactive_simulation.rst

```diff
@@ -0,0 +1,217 @@
+                                                 SIS_DiseaseModel, Risk, DirectEffect,
+                                                 MagicWandIntervention)
+
+    components = [BasePopulation(),
+                  Mortality(),
+                  SIS_DiseaseModel('diarrhea'),
+                  Risk('child_growth_failure'),
+                  DirectEffect('child_growth_failure', 'infected_with_diarrhea.incidence_rate'),
+                  DirectEffect('child_growth_failure', 'infected_with_diarrhea.excess_mortality_rate'),
+                  MagicWandIntervention('breastfeeding_promotion', 'child_growth_failure.proportion_exposed'),]
+
+Configurations
+~~~~~~~~~~~~~~
+We also need to create a dictionary of the configurations for the components.
+
+.. code-block:: python
+
+    config = {
+       'randomness': {
+           'key_columns': ['entrance_time', 'age'],
+       },
+       'population': {
+           'population_size': 10_000,
+       },
+       'diarrhea': {
+           'incidence': 2.5,        # Approximately 2.5 cases per person per year.
+           'remission': 42,         # Approximately 6 day median recovery time
+           'excess_mortality': 12,  # Approximately 22 % of cases result in death
+       },
+       'child_growth_failure': {
+           'proportion_exposed': 0.5,
+       },
+       'effect_of_child_growth_failure_on_infected_with_diarrhea.incidence_rate': {
+           'relative_risk': 5,
+       },
+       'effect_of_child_growth_failure_on_infected_with_diarrhea.excess_mortality_rate': {
+           'relative_risk': 5,
+       },
+       'breastfeeding_promotion': {
+           'effect_size': 0.5,
+       },
+    }
+
+Initialize
+~~~~~~~~~~
+
+We can setup and advance the simulation as we did above. The object we get back, an :func:`InteractiveContext <vivarium.interface.interactive.InteractiveContext>`,
+is the same no matter which helper function you use, though it may or may not have had its setup function called. Here,
+we will use :func:`setup_simulation() <vivarium.interface.interactive.setup_simulation>` to automatically initialize and setup our
+simulation context.
+
+.. code-block:: python
+
+    from vivarium.interface import setup_simulation
+
+    sim = setup_simulation(components, config)
+    sim.step()
+
+.. _progressing:
+
+Advancing the Simulation
+========================
+
+A simulation can be progressed in several ways, either in terms of steps of a size determined by the simulation
+configuration or in terms of the simulation's start and end time.  The simplest way to advance a simulation is to call
+:func:`run() <vivarium.interface.interactive.InteractiveContext.run>` on it, which will advance it from its specified
+start time to its specified end time.  Below is a table of the functions that can be called on an ``InteractiveContext``
+to advance a simulation in different ways.
+
+.. list-table:: InteractiveContext **functions for advancing simulations**
```

Comment:
the title ends up rendering slightly oddly since the first bit isn't bolded - is the first bit supposed to be a link?

Reply:
it was and I removed it, good call